### DISCUSSION TITLE
Issue with 2023-05 Cumulative Update Preview for Windows 11 Version 22H2 (KB5026446)

### DISCUSSION THREAD

**Discussion by arcyc** [valinet/ExplorerPatcher#1485] [General]
Issue with 2023-05 Cumulative Update Preview for Windows 11 Version 22H2 for x64-based Systems (KB5026446):

Properties > Start Menu > When using multiple displays, open Start ton this monitor when invoked using the keyboard: **Monitor on which the cursor is on** is not working properly

With Position on screen: **Center** it shows up in the middle of the 2nd monitor when cursor is on 2nd monitor
With Position on screen: **At screen edge** it shows up on the main monitor when cursor is on 2nd monitor

Note: this also causes an issue when this settings is the default settings - clicking the start menu with the mouse on the **second** screen when the Start Menu is centered causes it to be displayed half hidden off the bottom of the **primary** screen

**Comment by pyrates999** [comment]
Preview updates are not supported because microsoft doesn't provide symbols for them.  Uninstall any preview updates if you want to use explorer patcher.

source: https://github.com/valinet/ExplorerPatcher/wiki/Symbols
